File: python/program/Vehicle.py
class Vehicle:
    def __init__(self, plat, merk, tahun, warna):
        self.__plat = plat
        self.__merk = merk
        self.__tahun = tahun
        self.__warna = warna

    def get_plat(self):
        return self.__plat

    def set_plat(self, plat):
        self.__plat = plat

    def get_merk(self):
        return self.__merk

    def set_merk(self, merk):
        self.__merk = merk

    def get_tahun(self):
        return self.__tahun

    def set_tahun(self, tahun):
        self.__tahun = tahun

    def get_warna(self):
        return self.__warna

    def set_warna(self, warna):
        self.__warna = warna

File: LP4DPBO2024C1/program/Vehicle.cpp
#include "Vehicle.h"

Vehicle::Vehicle() : plat(""), merk(""), tahun(0), warna("") {}

Vehicle::Vehicle(string plat, string merk, int tahun, string warna) 
    : plat(plat), merk(merk), tahun(tahun), warna(warna) {}

void Vehicle::setPlat(string plat) {
    this->plat = plat;
}

string Vehicle::getPlat() const {
    return plat;
}

void Vehicle::setMerk(string merk) {
    this->merk = merk;
}

string Vehicle::getMerk() const {
    return merk;
}

void Vehicle::setTahun(int tahun) {
    this->tahun = tahun;
}

int Vehicle::getTahun() const {
    return tahun;
}

void Vehicle::setWarna(string warna) {
    this->warna = warna;
}

string Vehicle::getWarna() const {
    return warna;
}
